core/src/main/java/bisq/core/dao/node/full/network/FullNodeNetworkService.java

```diff
@@ -173,6 +189,24 @@ public void onFault(String errorMessage, @Nullable Connection connection) {
             } else {
                 log.warn("We have stopped already. We ignore that onMessage call.");
             }
+        } else if (networkEnvelope instanceof RepublishBlindVotesRequest) {
+            ObservableList<BlindVotePayload> appendOnlyStoreList = blindVoteListService.getAppendOnlyStoreList();
+            appendOnlyStoreList
+                    .forEach(blindVotePayload -> {
+                        // We want a random delay between 0.1 and 30 sec. depending on the number of items
```

Comment:
We don't want to stress the network with many broadcasts, so if there are many objects it will be delayed, the more objects the longer the dealy, but not more then 30 sec.